### PR TITLE
improve display and formatting of reproducescript examples

### PR DESCRIPTION
**Pull request by robertoostenveld**
FYI @matsvanes @schoffelen @Spaak, in this pull request I did the following 

- updated formatting and level of the headings
- rewrote some of the text
- resized figures
- add links to reference docu for main ft_ functions
- indented code correctly
- add links between the three pages

### CODE REVIEW

File: example/reproducescript_group.md [hunk 1]
```diff
@@ -295,474 +295,476 @@ To create a standard script from the analysis pipeline, the `ft_default` variabl
...
+
+Note that there are other strategies for improving shareability and reproducibility, and we don't claim that _reproducescript_ is the best way in every scenario. Rather, it is one of many tools that can aid the researcher to improve the community's standard in methodological transparency and robustness of results. For other strategies, we refer the reader to the pre-print in which we first described _reproducescript_.
 
 ## Suggested further reading
-- [Reducing the efforts to create reproducible analysis code with FieldTrip](https://doi.org/10.1101/2021.02.05.429886 )
+
+- [Reducing the efforts to create reproducible analysis code with FieldTrip](https://doi.org/10.1101/2021.02.05.429886)
 - [Reproducescript on a full study](/example/reproducescript_andersen)
```
Comment (matsvanes) [inline example/reproducescript_group.md:770]:
```suggestion
- [Using reproducescript on a full study](/example/reproducescript_andersen)
```

File: example/reproducescript.md [hunk 2]
```diff
@@ -1,178 +1,181 @@
...
+
+To explain the new reproducescript functionality, we will demonstrate its use it with a simple example pipeline for a single-subject analysis that comprises only a few analysis steps. Second, we demonstrate its application in a complete pipeline with preprocessing for multiple subjects, followed by a group analysis. The original idiosyncratic scripts that we selected for these first two examples are relatively clean and transparent, which means they are easily reproducible even without the new reproducescript functionality. As a final, third, example, we will apply _reproducescript_ to an already published analysis pipeline that contains more complexity, and thus benefits more from the _reproducescript_ functionality. The analysis code and data used in these examples are publicly available in the [Donders Repository](https://doi.org/10.34973/21pa-dg13).
```
Comment (matsvanes) [inline example/reproducescript.md:24]:
```suggestion
To explain the new reproducescript functionality, we will demonstrate its use with a simple example pipeline for a single-subject analysis that comprises only a few analysis steps. Second, we demonstrate its application in a complete pipeline with preprocessing for multiple subjects, followed by a group analysis. The original idiosyncratic scripts that we selected for these first two examples are relatively clean and transparent, which means they are easily reproducible even without the new reproducescript functionality. As a final, third, example, we will apply _reproducescript_ to an already published analysis pipeline that contains more complexity, and thus benefits more from the _reproducescript_ functionality. The analysis code and data used in these examples are publicly available in the [Donders Repository](https://doi.org/10.34973/21pa-dg13).

```

File: example/reproducescript.md [hunk 3]
```diff
@@ -1,178 +1,181 @@
...
 
-Many researchers are not formalled trained as software engineers or computer scientists. Thus, the quality, readability, and generalizability of analysis scripts is highly dependent on the individual researcher's coding style and expertise. Unfortunately, the variability in the quality of analysis scripts might compromise the reproducibility of results. Re-running the analysis pipeline might lead to different results, or the pipeline doesn't run on someone else's computer. 
+Many researchers are not formalled trained as software engineers or computer scientists. Thus, the quality, readability, and generalizability of analysis scripts is highly dependent on the individual researcher's coding style and expertise. Unfortunately, the variability in the quality of analysis scripts might compromise the reproducibility of results. Re-running the analysis pipeline might lead to different results, or the pipeline doesn't run on someone else's computer.
```
Comment (matsvanes) [inline example/reproducescript.md:18]:
```suggestion
Many researchers are not formally trained as software engineers or computer scientists. Thus, the quality, readability, and generalizability of analysis scripts is highly dependent on the individual researcher's coding style and expertise. Unfortunately, the variability in the quality of analysis scripts might compromise the reproducibility of results. Re-running the analysis pipeline might lead to different results, or the pipeline doesn't run on someone else's computer.

```

**Review by matsvanes — APPROVED** [review]
These changes all make sense to me. I'll check the formatting when it's updated on the website.

**Comment by robertoostenveld** [comment]
let's check in 10 minutes or so how it all looks.

**Pull request merged**